opencti-platform/opencti-front/src/private/components/data/connectors/Connector.jsx

```diff
@@ -404,17 +604,109 @@ const ConnectorComponent = ({ connector, relay }) => {
+                        {t_i18n('Not provided')}
+                      </Typography>
+                    </>)
+                  )
+                )}
+              </Grid>
+
               <Grid item xs={6}>
                 <Typography variant="h3" gutterBottom={true}>
-                  {t_i18n('Listen queue')}
+                  {t_i18n('Next run')}
                 </Typography>
-                <pre>{connector.config.listen}</pre>
+                {connector.connector_info && (
+                  // eslint-disable-next-line no-nested-ternary
+                  connector.connector_info.run_and_terminate ? (
+                    <Typography variant="body1" gutterBottom={true}>
+                      {t_i18n('External schedule')}
+                    </Typography>
+                  ) : (
+                    connector.connector_info.next_run_datetime !== null ? (
+                      <Typography variant="body1" gutterBottom={true}>
+                        {nsdt(connector.connector_info?.next_run_datetime)}
+                      </Typography>
+                    ) : (
+                      <Typography variant="body1" gutterBottom={true}>
+                        {t_i18n('Not provided')}
+                      </Typography>
+                    )
+                  )
+                )}
+                {!connector.connector_info && (
+                  <Typography variant="body1" gutterBottom={true}>
+                    {t_i18n('Not provided')}
+                  </Typography>
+                )}
               </Grid>
               <Grid item xs={6}>
-                <Typography variant="h3" gutterBottom={true}>
-                  {t_i18n('Push queue')}
+                <Typography variant="h3" gutterBottom={true} >
+                  {t_i18n('Server capacity')}
                 </Typography>
-                <pre>{connector.config.push}</pre>
+                { // eslint-disable-next-line no-nested-ternary
+                  connector.connector_info && connector.connector_info.queue_messages_size !== 0 ? (
+                    <FieldOrEmpty source={connector.connector_info?.queue_messages_size}>
+                      <span style={isBuffering() ? { color: theme.palette.warning.main } : {}}>{connector.connector_info?.queue_messages_size.toFixed(2)}</span>
+                      <span> / {connector.connector_info?.queue_threshold} Mo</span>
+                    </FieldOrEmpty>
+                  ) : (
+                    connector.connector_info && connector.connector_info.queue_messages_size === 0 && connector.connector_info.last_run_datetime
+                      ? (<FieldOrEmpty source={connector.connector_info?.queue_messages_size}>
+                        <span style={isBuffering() ? { color: theme.palette.warning.main } : {}}>{connector.connector_info?.queue_messages_size.toFixed(2)}</span>
+                        <span> / {connector.connector_info?.queue_threshold} Mo</span>
+                      </FieldOrEmpty>)
```

Comment:
I don't understand this whole condition : we display exactly the same thing if connector.connector_info.queue_messages_size === 0 or not, so why the condition ?

Reply:
yes, you're right, it's a mistake